terraform/datadog/etcd.tf

```diff
@@ -0,0 +1,23 @@
+resource "datadog_monitor" "etcd_one_leader" {
+  name                = "${format("%s etcd IsLeader count is exactly one (%s)", var.env, element(list("upper", "lower"), count.index))}"
+  type                = "query alert"
+  message             = "There is not exactly one etcd reporting IsLeader."
+  escalation_message  = "There is still not exactly one etcd reporting IsLeader."
+  notify_no_data      = false
+  require_full_window = true
+
+  count = 2
+
+  query = "${
+    format(
+      "min(last_5m):sum:cf.etcd.IsLeader{deployment:%s}.fill(last, 60) %s",
+      var.env,
+      element(list("> 1", "< 1"), count.index)
+    )}"
+
+  thresholds {
+    critical = "${element(list("1", "1"), count.index)}"
```

Comment:
Because this way is obvious how to define an lower/upper limit and that it needs to be updated. In this case the upper and lower limits are the same, but I think keeping it will ease reuse this pattern later. 

It also makes obvious that this critical value is related to the query expression.

BTW, if the `critical` value mismatches the value in the query for `<` or `>`, the datadog api call fails in a really obscure way.